test/call-cabal-project-to-nix/default.nix

```diff
@@ -9,6 +9,7 @@ let
     index-state = "2019-04-30T00:00:00Z";
     # reuse the cabal-simple test project
     src = ../cabal-simple;
+    hostPkgs = import pkgs.path { config = {}; overlays = []; };
```

Comment:
Shouldn't this be `buildPkgs`? 
Also isn't this the same as `pkgs.BuildPackages`, except for throwing away the overlays?

Reply:
the complication, is when your doing a fully native darwin build, but you want the x86 nixpkgs

darwin is technically not a cross-compile, so `.buildPackages` is likely going to still be darwin